pyeudiw/openid4vp/schema.py

```diff
@@ -41,3 +43,214 @@ class ResponseSchema(BaseModel):
+                                            ), ...),
+                          )
+
+
+class WalletInstanceAttestation(BaseModel):
+    """
+    Schema to validate a Wallet Instance Attestation. The attestation, in the form of a JWT header and payload,
+    has the properties listed below.
+        Header:
+            - alg
+            - typ
+            - kid
+            - x5c
+            - trust_chain
+        Payload:
+            - iss: The public url of the Wallet Instance attestation issuer.
+            - sub: Thumbprint value of the JWK of the Wallet Instance for which the attestation is being issued.
+            - iat: Unix timestamp of the issuance datetime.
+            - exp: Unix timestamp beyond which the presentation of the digital credential will no longer be considered
+                   valid.
+            - type: String: "WalletInstanceAttestation".
+            - policy_uri: URL to the privacy policy of the wallet.
+            - tos_uri: URL to the terms of use of the Wallet Provider.
+            - logo_uri: URL of the Wallet Provider logo in SVG format.
+            - attested_security_context:
+                Attested security context:
+                Represents a level of "trust" of
+                the service containing a Level Of
+                Agreement defined in the metadata
+                of the Wallet Provider.
+            - cnf:
+                This parameter contains the jwk
+                parameter
+                with the public key of the Wallet
+                necessary for the holder binding.
+            - authorization_endpoint:
+                URL of the OP's OAuth 2.0
+                Authorization Endpoint.
+            - response_types_supported:
+                JSON array containing a list of
+                the OAuth 2.0 response_type values
+                that this OP supports.
+            - vp_formats_supported:
+                JSON object containing
+                jwt_vp_json and jwt_vc_json
+                supported algorithms array.
+            - request_object_signing_alg_values_supported:
+                JSON array containing a list of the
+                JWS signing algorithms (alg values)
+                supported by the OP for Request Objects.
+            - presentation_definition_uri_supported:
+                Boolean value specifying whether the
+                Wallet Instance supports the transfer of
+                presentation_definition by
+                reference, with true indicating support.
+    """
+    header: create_model(header_model_name,
+                         alg=(Literal[
+                             "RS256",
+                             "RS384",
+                             "RS512",
+                             "ES256",
+                             "ES384",
+                             "ES512",
+                             "PS256",
+                             "PS384",
+                             "PS512",
```

Comment:
these should not be fixed in the code but taken from a configuration when the backed uses these schema, otherwise these should be taken from a constructor `__init__` if possible or any other strategy but get it our from the sources, these must be parameters (even with a default fallback as workaround)

Reply:
Resolved in [d0d3be0](https://github.com/italia/eudi-wallet-it-python/pull/71/commits/d0d3be073cc01b1cefaad86d5925358b6a982ddf)